codegen/generator/nodejs/templates/src/header.ts.tmpl

```diff
@@ -47,16 +47,19 @@ class BaseClient {
   /**
    * @hidden
    */
-  protected async _compute<T>() : Promise<Record<string, T>> {
+  protected async _compute<T>(): Promise<T> {
     try {
       // run the query and return the result.
       const query = queryBuilder(this._queryTree)
-      const computeQuery: Awaited<Promise<Record<string, T>>> =  await this.client.request(gql`${query}`)
+      const computeQuery: Awaited<T> = await this.client.request(
```

Comment:
Just for info/understanding: What did TypeScript put as the `string` of this `Record<string, T>`?

Reply:
It was based on GraphQl response